backend/src/validation/sophisticatedChatValidation.ts

```
@@ -4,96 +4,80 @@
 const messageSchema = z.string()
-  .min(1, 'Zpráva musí obsahovat alespoň 1 znak')
-  .max(10000, 'Zpráva může obsahovat maximálně 10000 znaků')
-  .refine((val) => val.trim().length > 0, 'Zpráva nemůže být prázdná');
+  .min(1, 'Zpráva nemůže být prázdná')
+  .max(10000, 'Zpráva může obsahovat maximálně 10000 znaků');
 
 // Conversation ID validation
 const conversationIdSchema = z.string()
   .uuid('ID konverzace musí být platné UUID');
+const conversationIdSchema = z.string()
+  .uuid('ID konverzace musí být platné UUID');
```

Biome (2.1.2)

[error] Shouldn't redeclare 'conversationIdSchema'. Consider to delete it or rename it.

'conversationIdSchema' is defined here:


(lint/suspicious/noRedeclare)